arch/arm64/src/common/arm64_gicv3.c

```diff
@@ -170,7 +196,7 @@ void arm64_gic_irq_enable(unsigned int intid)
   if (GIC_IS_SPI(intid))
     {
       putreg64(MPIDR_TO_CORE(GET_MPIDR()),
-               IROUTER(GET_DIST_BASE(intid), intid));
+              IROUTER(GET_DIST_BASE(intid), intid));
```

Comment:
revert?

Reply:
commit 0fd1de1ceaff0af1495ca3a9370dc102d146afc1
seem change this，please check